### PR TITLE
Latest translations

### PR DESCRIPTION
**Pull request by rgeraghty**
Latest translations:
- Updated @react-spectrum/picker for latest translations to match English "Select..."
- Updated @react-spectrum/s2 for latest partial translations

### CODE REVIEW

File: packages/@react-spectrum/picker/intl/ar-AE.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "loading": "جارٍ التحميل...",
-  "placeholder": "حدد خيارًا..."
+  "placeholder": "تحديد…"
```
Comment (rgeraghty) [inline packages/@react-spectrum/picker/intl/ar-AE.json:3]: English changed to just Select so translations updated to match

File: packages/@react-spectrum/s2/intl/ar-AE.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  "breadcrumbs.more": "More items",
   "button.pending": "قيد الانتظار",
```
Comment (rgeraghty) [inline packages/@react-spectrum/s2/intl/ar-AE.json:3]: Some locales are in English, which is ok, just have not recevied translation yet, will be updated once we get translation in

File: packages/@react-spectrum/s2/intl/ar-AE.json [hunk 3]
```diff
@@ -12,7 +13,7 @@
   "label.(optional)": "(اختياري)",
   "label.(required)": "(مطلوب)",
   "menu.moreActions": "المزيد من الإجراءات",
-  "picker.placeholder": "حدد خيارًا...",
+  "picker.placeholder": "تحديد…",
   "slider.maximum": "أقصى",
```
Comment (rgeraghty) [inline packages/@react-spectrum/s2/intl/ar-AE.json:17]: English string changed to shorter select, update translations to match (many locales)

File: packages/@react-spectrum/s2/intl/de-DE.json [hunk 4]
```diff
@@ -22,6 +23,6 @@
   "table.sortDescending": "Absteigend sortieren",
   "tag.actions": "Aktionen",
   "tag.hideButtonLabel": "Weniger zeigen",
-  "tag.noTags": "Keine",
+  "tag.noTags": "Kein(e)",
```
Comment (devongovett) [inline packages/@react-spectrum/s2/intl/de-DE.json:26]: ?

File: packages/@react-spectrum/s2/intl/et-EE.json [hunk 5]
```diff
@@ -1,18 +1,19 @@
...
+  "breadcrumbs.more": "More items",
   "button.pending": "ootel",
   "contextualhelp.help": "Spikker",
   "contextualhelp.info": "Teave",
   "dialog.alert": "Teade",
   "dialog.dismiss": "Lõpeta",
   "dropzone.replaceMessage": "Kukuta fail asendamiseks",
   "inlinealert.informative": "Teave",
-  "inlinealert.negative": "Tõrge",
+  "inlinealert.negative": "Viga",
```
Comment (devongovett) [inline packages/@react-spectrum/s2/intl/et-EE.json:10]: Does this need to change in `@react-spectrum/inlinealert/intl/et-EE.json` too? This was originally copied from there

File: packages/@react-spectrum/s2/intl/fr-FR.json [hunk 6]
```diff
@@ -22,6 +23,6 @@
   "table.sortDescending": "Trier par ordre décroissant",
   "tag.actions": "Actions",
   "tag.hideButtonLabel": "Afficher moins",
-  "tag.noTags": "Aucun",
+  "tag.noTags": "Aucun(e)",
```
Comment (devongovett) [inline packages/@react-spectrum/s2/intl/fr-FR.json:26]: ?

File: packages/@react-spectrum/s2/intl/it-IT.json [hunk 7]
```diff
@@ -7,12 +8,12 @@
   "dropzone.replaceMessage": "Rilascia il file da sostituire",
   "inlinealert.informative": "Informazioni",
   "inlinealert.negative": "Errore",
-  "inlinealert.notice": "Avviso",
+  "inlinealert.notice": "Avvertenza",
```
Comment (devongovett) [inline packages/@react-spectrum/s2/intl/it-IT.json:11]: Same - for `packages/@react-spectrum/inlinealert/intl/it-IT.json`

File: packages/@react-spectrum/s2/intl/nb-NO.json [hunk 8]
```diff
@@ -12,16 +13,16 @@
   "label.(optional)": "(valgfritt)",
   "label.(required)": "(obligatorisk)",
   "menu.moreActions": "Flere handlinger",
-  "picker.placeholder": "Velg et alternativ...",
+  "picker.placeholder": "Velg …",
```
Comment (devongovett) [inline packages/@react-spectrum/s2/intl/nb-NO.json:16]: Space before ellipses was removed below but added here?

Comment (rgeraghty) [reply packages/@react-spectrum/s2/intl/nb-NO.json:16]: This is consistent with other "Velg …" so used here. For this PR will keep but do overall review with vendor to fix these for future update. For example, there are cases of:
- Laster inn ...
- Laster inn...

I need to check with translators as almost 50% of each type

File: packages/@react-spectrum/s2/intl/fr-FR.json (hunk 6 above)
Comment (rgeraghty) [reply packages/@react-spectrum/s2/intl/fr-FR.json:26]: I'm reverting this, change leveraged from another app, but no linguistic reason for change in React-Spectrum

File: packages/@react-spectrum/s2/intl/de-DE.json (hunk 4 above)
Comment (rgeraghty) [reply packages/@react-spectrum/s2/intl/de-DE.json:26]: I'm reverting this, change leveraged from another app, but no linguistic reason for change in React-Spectrum

File: packages/@react-spectrum/s2/intl/et-EE.json (hunk 5 above)
Comment (rgeraghty) [reply packages/@react-spectrum/s2/intl/et-EE.json:10]: Yes I'll change other occurrence as well, Viga is most used. I'll also do followup with vendor why we have both,

File: packages/@react-spectrum/s2/intl/it-IT.json (hunk 7 above)
Comment (rgeraghty) [reply packages/@react-spectrum/s2/intl/it-IT.json:11]: I've updated to "Alert" are consistent and "Warning" are consistent, and will do a followup review with vendor

**Comment by rgeraghty** [comment]
Thanks @devongovett I've updated some translations and added comments,

**Review by devongovett — APPROVED** [review]

**Review by reidbarber — APPROVED** [review]

**Review by dannify — APPROVED** [review]

**Pull request merged**